exercises/saddle-points/SaddlePointsTest.cs

```diff
@@ -0,0 +1,75 @@
+// This file was auto-generated based on version 1.0.0 of the canonical data.
+
+using Xunit;
+using System;
+
+public class SaddlePointsTest
+{
+    [Fact]
+    public void Can_identify_single_saddle_point()
+    {
+        var input = new[,]
+        {
+        	{ 9, 8, 7 },
```

Comment:
The indentation is slight off (also in the other cases), I would expect the following:

```csharp
var input = new[,]
{
    { 9, 8, 7 },
};
```